flink-table-store-format/src/main/java/org/apache/flink/table/store/format/orc/OrcFileFormat.java

```diff
@@ -115,4 +124,96 @@ private static Properties getOrcProperties(ReadableConfig options) {
         properties.forEach((k, v) -> orcProperties.put(IDENTIFIER + "." + k, v));
         return orcProperties;
     }
+
+    private static LogicalType convertLogicalType(LogicalType type) {
```

Comment:
can we change a method name? or add some comments to explain `BINARY` and `VARBINARY`